app/assets/javascripts/discourse/app/controllers/users.js

```diff
@@ -8,7 +8,7 @@ export default Controller.extend({
   application: controller(),
   queryParams: ["period", "order", "asc", "name", "group", "exclude_usernames"],
   period: "weekly",
-  order: "likes_received",
+  order: "username",
```

Comment:
Why does it change the default to username? I feel likes_received is a good default.

Reply:
@eviltrout I didn't find any specs related to it or around to it in the core

can u please help me, like what a spec should contain for this pr?